src/main/java/edu/harvard/iq/dataverse/api/auth/BearerTokenAuthMechanism.java

```diff
@@ -0,0 +1,116 @@
+package edu.harvard.iq.dataverse.api.auth;
+
+import com.nimbusds.oauth2.sdk.ParseException;
+import com.nimbusds.oauth2.sdk.token.BearerAccessToken;
```

Comment:
Online I see it called "bearer token", not "bearer access token" and I was wondering where the former comes from. I supposed it's this BearerAccessToken class.

Should we consider switching to "bearer token" where we can in our own code (throughout this pull request, I mean)? That it, drop "access" from the middle? Shorter. In line with the RFC which has "Bearer Token Usage" in the title: https://datatracker.ietf.org/doc/html/rfc6750

I looked for any discussion about this upstream but didn't find any: https://bitbucket.org/connect2id/oauth-2.0-sdk-with-openid-connect-extensions/src/master/

Reply:
Oauth2 and OIDC define and use different kinds of Token, AcessToken is one of them. I assume its come from there  (https://oauth.net/id-tokens-vs-access-tokens/). Nevertheless, I shortened the constants names and strings